src/main/java/com/google/cloud/tools/appengine/configuration/RunConfiguration.java

```diff
@@ -179,9 +179,14 @@ public String getPythonStartupArgs() {
     return pythonStartupArgs;
   }
 
-  @Nullable
+  /**
+   * Returns a mutable list that exposes the internal state.
```

Comment:
```suggestion
   * Returns JVM flags as a mutable list that exposes the internal state.
```

But I believe the implementation does not expose the mutable internal state any longer.

Reply:
done